package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ecommerce-app",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www",
+    "db:makemigrations": "./node_modules/sequelize-auto-migrations/bin/makemigration.js"
+  },
+  "dependencies": {
+    "basic-auth": "^2.0.1",
+    "connect-session-sequelize": "^6.1.1",
+    "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
+    "debug": "^2.6.9",
+    "dotenv": "^8.2.0",
+    "express": "^4.16.4",
+    "express-session": "^1.17.0",
+    "http-errors": "^1.6.3",
+    "jade": "^1.11.0",
+    "jsonwebtoken": "^8.5.1",
+    "morgan": "^1.9.1",
+    "pg": "^7.18.2",
+    "pg-hstore": "^2.3.3",
+    "promise": "^8.1.0",
+    "sequelize": "^5.21.5",
+    "sequelize-auto-migrations": "github:scimonster/sequelize-auto-migrations#a063aa6535a3f580623581bf866cef2d609531ba",
```

Comment:
Why this package?

Reply:
well that's a script i found on the internet which automatically write  migrations based on your models and the _current.json is for keeping track of the migrations. As you know we wrote models first instead of following the ususal sequelize-cli way so i used this script instead of reinventing the wheel to create migrations automatically and follow the usual procedure for the next tables i wanted to create